docker/release/Dockerfile

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
Move to root, as discussed (and required by https://devcenter.heroku.com/articles/build-docker-images-heroku-yml#known-issues-and-limitations).